content/en/blog/2023/waypoint-proxy-made-simple/index.md

```diff
@@ -0,0 +1,221 @@
+
+{{< image width="70%"
+    link="sidecar-config.png"
+    caption="Every sidecar has configuration about all other sidecars"
+    >}}
+
+In the sidecar model, we have 4 workloads, each with 4 sets of configuration. If any of those configurations changed, all of them would need to be updated. In total there are 16 configurations distributed.
+
+In the waypoint architecture, however, the configuration is dramatically simplified:
+
+{{< image width="70%"
+    link="waypoint-config.png"
+    caption="Each waypoint only has configuration for its own namespace"
+    >}}
+
+Here, we see a very different story. We have only 2 waypoint proxies, as each one is able to serve the entire namespace, and each one only needs configuration for its own namespace. In total we have 25% of the amount of configuration sent, even for a simple example.
+
+If we scale each namespace up to 25 deployments with 10 pods each and each waypoint deployment with 2 pods for high availability, the numbers are even more impressive - the waypoint config distribution requires just 0.8% of the configuration distribution of the sidecar, as the table below illustrates!
+
+| Config Distribution         |         Namespace 1              |       Namespace 2                |     Total     |
+| --------------------------- | -------------------------------- | -------------------------------- | ------------- |
+| Sidecars                    | 25 configurations * 250 sidecars | 25 configurations * 250 sidecars |    12500      |
+| Waypoints                   | 25 configurations * 2 waypoints  | 25 configurations * 2 waypoints  |     100       |
+| Waypoints / Sidecars        |              0.8%                |               0.8%               |      0.8%     |
+
+While we use namespace scoped waypoint proxies to illustrate the simplification above, the simplification is similar
+when you apply it to service account waypoint proxies.
+
+This reduced configuration means lower resource usage (CPU, RAM, and network bandwidth) for both the
+control plane and data plane. While users today can see similar improvements with careful usage of
+`exportTo` in your Istio networking resources or the [Sidecar](/docs/reference/config/networking/sidecar/) API,
+in ambient mode this is no longer required, making scaling a breeze.
+
+## What if my destination doesn’t have a waypoint proxy?
+
+The design of ambient mode centers around the assumption that most configuration is best implemented by the service producer, rather than the service consumer. However, this isn't always the case - sometimes we need to configure traffic management for destinations we don't control. A common example of this would be connecting to an external service with improved resilience to handle occasional connection issues (e.g. to add a timeout for calls to `example.com`).
```

Comment:
```suggestion
The design of ambient mode centers around the assumption that most configuration is best implemented by the service producer, rather than the service consumer. However, this isn't always the case - sometimes we need to configure traffic management for destinations we don't control. A common example of this would be connecting to an external service with improved resilience to handle occasional connection issues (e.g., to add a timeout for calls to `example.com`).
```